README.md

```diff
@@ -89,7 +89,10 @@ def hello_world_plug(conn, _opts) do
 end
 ```
 
-A module plug implements an `init/1` function to initialize the options and a `call/2` function which receives the connection and initialized options and returns the connection:
+A module plug implements two functions:
+
+1. An `init/1` function that initializes and returns the plug options.
+2. A `call/2` function which receives the connection and the value returned by `init/1`, and returns the connection:
```

Comment:
I am leaning on 👎 for this change. The README should be as concise as possible and mostly "hint" and what Plug is and can do, so the more informal and less documentation-like we can keep it, the happier I would be :)

Reply:
The original sentence is

> A module plug implements an `init/1` function to initialize the options and a `call/2` function which receives the connection and initialized options and returns the connection

Are you fine with it as is?

Reply:
Yeah, I'd prefer the one we had before.